tests/serializers.py

```diff
@@ -96,40 +96,71 @@ class Meta:
             'members',
             'users',
             'loc1users',
-            'loc1usersLambda'
+            'loc1usersLambda',
+            'loc1usersGetter',
         )
 
     permissions = DynamicRelationField(
         'PermissionSerializer',
         many=True,
         deferred=True)
+
+    # Infer serializer from source
     members = DynamicRelationField(
-        'UserSerializer',
         source='users',
         many=True,
-        deferred=True)
+        deferred=True
+    )
 
     # Intentional duplicate of 'users':
     users = DynamicRelationField(
-        'UserSerializer',
         many=True,
-        deferred=True)
+        deferred=True
+    )
 
-    # For testing default queryset on relations:
+    # Queryset for get filter
     loc1users = DynamicRelationField(
         'UserSerializer',
         source='users',
         many=True,
         queryset=User.objects.filter(location_id=1),
-        deferred=True)
+        deferred=True
+    )
 
+    # Dynamic queryset through lambda
     loc1usersLambda = DynamicRelationField(
         'UserSerializer',
         source='users',
         many=True,
         queryset=lambda srlzr: User.objects.filter(location_id=1),
         deferred=True)
 
+    # Custom getter/setter
+    loc1usersGetter = DynamicRelationField(
+        'UserSerializer',
+        source='*',
```

Comment:
Source = '*' generally means "we don't know what the source is, there's some overrides we are going to rely on instead", e.g. this already happens by default with `DynamicMethodField`